Update readme (#30)
T-frog/tf-2md3-firmware: README.md

Contributer: Atsushi Watanabe at Tsukuba University

## Compile Firmware

### Setting Development Environment

This is a quick guide which contains all the tools you'll need to install on your development environment.

* samba
```
$ git clone https://github.com/at-wat/samba.git
$ cd samba
$ ./configure
$ make
$ sudo make install
```

* tf2md3_flash
```
$ cd ../
$ git clone https://github.com/T-frog/tf2md3_flash.git
$ cd tf2md3_flash
$ sudo make install
```

* gcc-arm-none-eabi
```

> $ wget https://developer.arm.com/-/media/Files/downloads/gnu-rm/6-2017q2/gcc-arm-none-eabi-6-2017-q2-update-linux.tar.bz2?product=GNU%20ARM%20Embedded%20Toolchain,64-bit,,Linux,6-2017-q2-update > gcc-arm-none-eabi.tar.bz2

Reviewer: `wget` doesn't output contents to stdout by default. Redirecting to the file doesn't work.

$ sudo tar xjf gcc-arm-none-eabi.tar.bz2 --strip-components 1 -C /usr
```

### Compile

```
$ cd tfrog-motordriver
$ make
```

## Flash Firmware

**DEVICE_PATH**: Full path to a mounted device
**FIRMWARE**: A path to a compiled firmware

```
$ tf2md3_flash DEVICE_PATH FIRMWARE.bin
```

## LICENSE

> Deliverables of tf-2md3-firmware code are inherited from at91lib license as below.

Reviewer: I'm planning to update copyright of non-at91lib part because ATMEL's license looks not for inheriting.
(https://github.com/T-frog/tf-2md3-firmware/compare/update-copyright)

```suggestion
Some part of tf-2md3-firmware code are licensed under the at91lib license as below.
```
would be good for now.
